### PR TITLE
[WIP] GitHub Pages site update

### PR DESCRIPTION
**Pull request by Leland**
SUMMARY: Infrastructure "Modernize project main page."

Overhaul of the design and UX of the new CDDA site. I worked to style it similarly to the CDDA Discourse theme we're currently using. Have a look at what I've done with the place: https://leland.github.io/Blah/

### To-do:

- [x] Main styles
- [x] Add screenshots
- [x] Add SEO tags
- [ ] Style downloads section
- [ ] Style footer
- [ ] Separate home/page layouts
- [ ] Re-add "Contributions" and "Releases"

### CODE REVIEW

**Comment by Soyweiser** [comment]
Well atm the releases page is more prominent than the download one (where you can get the experimental version) so that should be made a little bit more clear. But we shouldn't just remove the releases at all imho. (it does provide a nice way of seeing how far the project has come).

Looks good btw. nice touch in picking pretty varied screenshots, with the different stuff seen (one with self aware, and being on fire, showing those interfaces, other with the npcs and monsters).

**Comment by Leland** [comment]
> Well atm the releases page is more prominent than the download one (where you can get the experimental version) so that should be made a little bit more clear. But we shouldn't just remove the releases at all imho. (it does provide a nice way of seeing how far the project has come).

Yup, completely agreed. I'm going to be sprucing up the download section, adding links to download the current Release version alongside the current experimental ones. I'll most likely add a "Releases" changelog link there.

> nice touch in picking pretty varied screenshots

Thanks dude! That was my intent. Tried to show some different things without getting spoiler-y. I may add a few more and may possibly move it to a slider as well.

**Comment by codemime** [comment]
> I may add a few more and may possibly move it to a slider as well.

Deathmobiles (or any vehicles whatsoever), they worth a glimpse.

**Comment by ZhilkinSerg** [comment]
Add some gif animation maybe?

**Comment by Leland** [comment]
I'll add a couple more screenshots but future ones can just be added via PR :)

**Comment by ZhilkinSerg** [comment]
Shouldn't http://www.cataclysmdda.org/ redirect to https://cataclysmdda.org/?

File: _includes/head.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  {% seo %}
+  <link rel="stylesheet" href="{{ "/assets/main.css" | relative_url }}">
+  <link rel="shortcut icon" type="image/png" href="assets/favicon.png">
+  {% feed_meta %}
+  {% if jekyll.environment == 'production' and site.google_analytics %}
+    {% include google-analytics.html %}
+  {% endif %}
```
Comment (BorkBorkGoesTheCode) [inline _includes/head.html]: Is Google Analytics necessary?

Comment (Leland) [reply _includes/head.html]: Well, it's not configured at all so it wouldn't be on. But I highly recommend that we do configure at some point, anonymized analytics are awesome

**Comment by Faalagorn** [comment]
And what about http://www.cataclysmdda.com/ – is it still being used?

**Comment by kevingranade** [comment]
No, that's a dead site that will either go away, or if I acquire the domain it'll redirect to cataclysmdda.org

**Comment by ZhilkinSerg** [comment]
Releases page (https://cataclysmdda.org/releases/) should be update with Android Tiles build download link (http://dev.narc.ro/cataclysm/jenkins-latest/Android/Tiles/)

**Comment by kevingranade** [comment]
Idle for ~half a year, feel free to reopen.

**Pull request closed without merging**